### PR TITLE
ci: run tests on PRs to typescript-branch

### PR DESCRIPTION
**Pull request by yunchipang**
## Description

**Issue(s) addressed**

- Resolves #

**What kind of change(s) does this PR introduce?**

- [x] Enhancement
- [ ] Bug fix
- [ ] Refactor

**Please check if the PR fulfils these requirements**

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Issue

**What is the current behavior?**

**What is the new behavior?**

## Breaking change

**Does this PR introduce a breaking change?**

### CODE REVIEW

File: .github/workflows/treetracker-wallet-api-pull-request-ci.yml [hunk 1]
```diff
@@ -55,8 +55,11 @@ jobs:
           DATABASE_URL: postgresql://postgres:postgres@postgres/postgres
 
       - name: run test
-        run: npm run test
+        run: npm test
         working-directory: ${{ env.project-directory }}
         env:
-          DATABASE_URL: postgresql://postgres:postgres@postgres/postgres
+          DATABASE_URL: postgresql://wallet_user:postgres@localhost:5432/wallet_user
```
Comment (Kpoke) [inline .github/workflows/treetracker-wallet-api-pull-request-ci.yml:61]: Revert this database URL change

File: src/modules/app/app.module.ts [hunk 2]
```diff
@@ -21,6 +21,7 @@ import { SeedService } from '../seed/seed.service';
     TypeOrmModule.forRoot({
       type: 'postgres',
       url: process.env.DATABASE_URL,
+      schema: process.env.DATABASE_SCHEMA || 'wallet',
```
Comment (yunchipang) [inline src/modules/app/app.module.ts:24]: @Kpoke i added the env variable `DATABASE_SCHEMA` here which is defined in the workflow yml file. however the github action is still not working🥲

File: src/modules/app/app.module.ts [hunk 3]
```diff
@@ -18,11 +18,16 @@ import { SeedService } from '../seed/seed.service';
...
-      entities: [ApiKey, Wallet, Event],
-      synchronize: true, // set to false in production
+    TypeOrmModule.forRootAsync({
+      useFactory: async () => {
+        return {
+          type: 'postgres',
+          url: process.env.DATABASE_URL,
+          schema: process.env.DATABASE_SCHEMA || undefined,
+          entities: [ApiKey, Wallet, Event],
+          synchronize: true, // Disable in production
```
Comment (Kpoke) [inline src/modules/app/app.module.ts:28]: Let's set this to false altogether and rely on migrations alone @yunchipang

Comment (yunchipang) [reply src/modules/app/app.module.ts:28]: @Kpoke do i submit another PR to set this to false?

**Pull request merged**